app/battle_implement/implementation.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from app.entities.knight import Knight
+from app.entities.stuff import Weapon, ArmourPart, SetArmourScope, Potion
+
+
+def crete_armour(armours: list) -> SetArmourScope | list:
+    if armours:
+        armour_parts = [ArmourPart(armour.get("part"),
+                                   armour.get("protection"))
+                        for armour in armours]
```

Comment:
fix indentation here, it has bad readability. Use something like this

![image](https://user-images.githubusercontent.com/104218550/219123286-3b61a794-8ca1-45ef-bdcb-3d4fe383fbf5.png)